src/terminal/node/terminal-backend-module.ts

```diff
@@ -22,13 +22,20 @@ export default new ContainerModule(bind => {
 @injectable()
 class TerminalExpressContribution implements BackendApplicationContribution {
     private terminals: Map<number, any> = new Map()
-    private logs: string[] = []
+
+    getShellExecutablePath() {
```

Comment:
Right this should be private will fix, yeah I'd rather not use a get property however since it's not a property of the object